docx_editor_mcp/cache.py

```
@@ -0,0 +1,185 @@
+        """
+        if explicit_author:
+            self._session_author = explicit_author
+            return explicit_author, False
+
+        if self._session_author:
+            return self._session_author, False
+
+        # First time - use system default
+        try:
+            default = getpass.getuser()
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)